### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/Level/leveldown/issues/396

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -305,48 +305,48 @@ Safety
...
+`leveldown` is an **OPEN Open Source Project**. This means that:
 
 > Individuals making significant and valuable contributions are given commit-access to the project to contribute as they see fit. This project is more like an open wiki than a standard guarded open source project.
 
 See the [contribution guide](https://github.com/Level/community/blob/master/CONTRIBUTING.md) for more details.
 
 ### Windows
 
-A large portion of the Windows support comes from code by [Krzysztof Kowalczyk](http://blog.kowalczyk.info/) [@kjk](https://twitter.com/kjk), see his Windows LevelDB port [here](http://code.google.com/r/kkowalczyk-leveldb/). If you're using LevelUP on Windows, you should give him your thanks!
+A large portion of the Windows support comes from code by [Krzysztof Kowalczyk](http://blog.kowalczyk.info/) [@kjk](https://twitter.com/kjk), see his Windows LevelDB port [here](http://code.google.com/r/kkowalczyk-leveldb/). If you're using `leveldown` on Windows, you should give him your thanks!
```
Comment (ralphtheninja) [inline README.md]: Yes, it should be `leveldown` and not `levelup` :)

File: README.md [hunk 2]
```diff
@@ -24,27 +24,27 @@ LevelDOWN
 Introduction
 ----------------------------
 
-LevelDOWN was extracted from [LevelUP](https://github.com/level/levelup) and now serves as a stand-alone binding for LevelDB.
+This module was originally part of [`levelup`](https://github.com/level/levelup) but was later extracted and now serves as a stand-alone binding for LevelDB.
```
Comment (ralphtheninja) [inline README.md]: I rewrote this to get a capital letter. Not really sure if we should keep this or not, do people ending up on leveldown really care about the lore and why it was extracted? If so, should that be the first sentence?

Comment (ralphtheninja) [reply README.md]: That said, I'm fine with keeping it as well.

File: README.md [hunk 3]
```diff
@@ -1,38 +1,50 @@
-LevelDOWN
+leveldown
 =========
 
-<img alt="LevelDB Logo" height="100" src="http://leveldb.org/img/logo.svg">
-
-**A Low-level Node.js LevelDB binding**
+> A low level node.js LevelDB binding
```
Comment (vweevers) [inline README.md]: - Why lowercase Node.js?
- low level => low-level (I think)

File: README.md [hunk 4]
```diff
@@ -1,38 +1,50 @@
...
-[![AppVeyor build status](https://img.shields.io/appveyor/ci/Level/leveldown.svg)](https://ci.appveyor.com/project/Level/leveldown)
-[![dependencies](https://david-dm.org/Level/leveldown.svg)](https://david-dm.org/level/leveldown) [![Greenkeeper badge](https://badges.greenkeeper.io/Level/leveldown.svg)](https://greenkeeper.io/)
-
-[![NPM](https://nodei.co/npm/leveldown.png?stars&downloads&downloadRank)](https://nodei.co/npm/leveldown/) [![NPM](https://nodei.co/npm-dl/leveldown.png?months=6&height=3)](https://nodei.co/npm/leveldown/)
+[![Travis](https://img.shields.io/travis/Level/leveldown.svg?label=travis)](http://travis-ci.org/Level/leveldown)
+[![AppVeyor](https://img.shields.io/appveyor/ci/Level/leveldown.svg?label=appveyor)](https://ci.appveyor.com/project/Level/leveldown)
+[![dependencies](https://david-dm.org/Level/leveldown.svg)](https://david-dm.org/level/leveldown)
+[![npm](https://img.shields.io/npm/dm/leveldown.svg)](https://www.npmjs.com/package/leveldown)
+
+  * <a href="#intro">Introduction</a>
```
Comment (vweevers) [inline README.md]: 👍

File: README.md (hunk 2 above)
Comment (vweevers) [reply README.md]: > Not really sure if we should keep this or not, do people ending up on leveldown really care about the lore and why it was extracted?

Nah. I think it's fine to remove the first sentence. (The TOC isn't that long here so it's close enough to the short description)

File: README.md [hunk 5]
```diff
@@ -293,46 +305,48 @@ Safety
...
+`leveldown` exposes a feature of LevelDB called [snapshots](http://leveldb.googlecode.com/git-history/f779e7a5d89f853fc5224f9eb8103ca2b8f2f555/doc/index.html). This means that when you do e.g. `createReadStream` and `createWriteStream` at the same time, any data modified by the write stream will not affect data emitted from the read stream. In other words, a LevelDB Snapshot captures the latest state at the time the snapshot was created, enabling the snapshot to iterate or read the data without seeing any subsequent writes. Any read not performed on a snapshot will implicitly use the latest state.
```
Comment (vweevers) [inline README.md]: Let's update the link to https://github.com/google/leveldb/blob/master/doc/index.md#snapshots

File: package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "leveldown",
-  "description": "A Node.js LevelDB binding, primary backend for LevelUP",
+  "description": "A node.js LevelDB binding, primary backend for levelup",
```
Comment (vweevers) [inline package.json]: Do we really need "Node.js" in the description of an npm package?

File: README.md (hunk 3 above)
Comment (ralphtheninja) [reply README.md]: No reason, other than I think it looks bad. Updating.

File: package.json (hunk 6 above)
Comment (ralphtheninja) [reply package.json]: Not really. But maybe it should be the same as the short descr in the readme.

File: README.md (hunk 5 above)
Comment (ralphtheninja) [reply README.md]: Done

File: package.json (hunk 6 above)
Comment (vweevers) [reply package.json]: Alright

**Comment by vweevers** [comment]
@ralphtheninja is this ready?

Also I was thinking, maybe we should just remove short descriptions from readme's that have an intro section (leveldown and levelup). In other words the first paragraph of the intro ís the short description.

**Comment by ralphtheninja** [comment]
Yep, ready.

**Comment by ralphtheninja** [comment]
Squashed and merged.

**Pull request merged**